travis/install_protobuf.sh

```diff
@@ -11,9 +11,9 @@ if [ "$protobuf_dist" != "" ]; then
   cd $protobuf_dist
 fi
 
-wget https://github.com/google/protobuf/archive/v3.0.0-beta-1.tar.gz
-tar -xvzf v3.0.0-beta-1.tar.gz
-cd protobuf-3.0.0-beta-1
+wget https://github.com/google/protobuf/archive/v3.0.0-beta-2.tar.gz
```

Comment:
Did it not work on OS X with beta-1? Unfortunately, I'm going to have to change this back to beta-1, because grpc is not yet compatible with protobuf v3.0.0-beta-2 due to https://github.com/grpc/grpc/issues/4505.